packages/manager/modules/vrack/src/dashboard/vrack.controller.js

```diff
@@ -236,7 +242,9 @@ export default class VrackMoveDialogCtrl {
           allServices.dedicatedServerInterface = [];
         }
 
-        return allServices;
+        return pickBy(allServices, (_, serviceType) => {
+          return this.features.isFeatureAvailable(FEATURE_NAMES[serviceType]);
+        });
```

Comment:
May be we can create a function to avoid duplicate snippet code:

```suggestion
        return this.getAvailableServices();
```

Reply:
This is done